Calculator/Program.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using CalculatorLibrary;
+using Newtonsoft.Json;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            bool endApp = false;
+            // Display title as the C# console calculator app.
+            Console.WriteLine("Console Calculator in C#\r");
+            Console.WriteLine("------------------------\n");
+
+            CalculatorProgram calculatorProgram = new CalculatorProgram();
+
+            while (!endApp)
+            {
+                // Declare variables and set to empty.
+                string numInput1 = "";
+                string numInput2 = "";
+                double result = 0;
+                string histResult;
+
+                Console.WriteLine("What would you like to do?");
+                Console.WriteLine("\tv - View History");
+                Console.WriteLine("\to - Calculator");
+                Console.WriteLine("\te - Exit");
+                Console.Write("Your option?");
+                string opt = Console.ReadLine();
+                if (opt == "v")
+                {
+                    // Calling the History Method
+                    calculatorProgram.EndHistory();
+                    Console.Write("Do you want to clear the history? Press 'y' to clear the history:");
+                    string clearHistory = Console.ReadLine();
+                    if (clearHistory == "y")
+                    {
+                        calculatorProgram.ClearHistory();
+                    }
+                }
+                else if (opt == "e")
+                {
+                    endApp = true;
+                }
+                else
+                {
+                    // Ask the user to type the first number.
+                    Console.Write("Type a number, and then press Enter or press 'h' to use history: ");
+                    numInput1 = Console.ReadLine();
+
+                    // using history
+                    if (numInput1 == "h")
+                    {
+                        histResult = calculatorProgram.UseHistory(numInput1);
+                        numInput1 = histResult;
+                    }                    
+
+                    double cleanNum1 = 0;
+                    while (!double.TryParse(numInput1, out cleanNum1))
+                    {
+                        Console.Write("This is not valid input. Please enter an integer value: ");
+                        numInput1 = Console.ReadLine();
+                    }
```

Comment:
Can be it's own method. Can also reuse this new method to get the second number and remove the code for that.